cfn-templates/cid-cfn.yml

```diff
@@ -4,67 +4,99 @@ Metadata:
   AWS::CloudFormation::Interface:
     ParameterGroups:
       - Label:
-          default: 'Prerequisites'
+          default: 'Common Parameters'
         Parameters:
           - PrerequisitesQuickSight
-          - PrerequisitesCUR
           - PrerequisitesQuickSightPermissions
-      - Label:
-          default: 'Mandatory parameters'
-        Parameters:
           - QuickSightUser
-          - CURBucketPath
-          - CURIsManagedByCloudFormation
       - Label:
-          default: Available dashboards. Keep all 'no' for the first deployment. Go to QuickSight, change Permission and after that update the stack putting to 'yes' 3 first dashboards.
+          default: CUDOS Cost-Intelligence-Dashboard and KPI-Dashboard. Require deployment of CUR via CloudFormation (cur-aggregation.yaml) or manually 24h before.
```

Comment:
24 hours is a not a hard requirement, I'd suggest 24 hours mention and add link to CUR dashboards deployment page, similar as for COD and TAO below

Reply:
What link to CUR deployment? AFIK we do not have that as a separate page. we have this -> https://wellarchitectedlabs.com/cost/200_labs/200_cloud_intelligence/cost-usage-report-dashboards/dashboards/deploy_dashboards/ this looks too long to me.